### PR TITLE
Feature: Add RCS control and movement to shuttles #3889

### LINKED ISSUE #3889: Feature: Add RCS control and movement to shuttles: TB: Budget Sprint 1
## Criteria:

- Implement the sprites with required functionality to the ShuttleControl NetTab that are created in #3888 
- Create the shuttle RCS thruster prefab with the sprites from #3888 
- Create the scripts to operate the RCS thruster from the Shuttle Console
- Create a particle system for the thruster itself and make it burst out gases when the player moves the shuttle via RCS. Also add soundfx
- When the RCS button is clicked the players WASD controls will now move the shuttle left, right, up, down one tile at a time and rather slowly
- When the player clicks the RCS button again then return the controls back to the player movement
- Have the ShuttleConsole check for RCS engines attached to the shuttle matrix and determine their direction
- Disallow direction in any direction that doesn't have an RCS engine or for any engines that have been badly damaged or broken off
- Make sure everything works in MP and test locally in headless mode

------------
### Time Based Bounty:

This is a time based bounty. Only one developer can work on a time based bounty at a time, unless you are collaborating. To claim that you are working on this job please talk to Doobly and you will be given instructions on what to do next. Current time based bounties are locked in to $11usd/hour as we have a small budget. We hope this will increase over time.

------------

### PR DESCRIPTION
**Pull request by michalczemierowski**
### Purpose
Fixes #3889

### Instructions for placing RCS on shuttle:
RCS thruster prefab path - **Assets/Resources/Prefabs/Objects/Shuttle/Vernor RCS.prefab**
#### RCS THRUSTER SETTINGS
Directional.InitialDirection should be the same as shuttle  MatrixMove.InitialFacing
ss
and DirectionalRotatesParent.PrefabChildrenOrientation should be the direction of RCS in local space.

> BOW (FRONT) = **UP**
> STERN (BACK) = **DOWN**
> PORT (LEFT) = **RIGHT**
> STARBOARD (RIGHT) = **LEFT**

ss2

### CODE REVIEW

File: UnityProject/Assets/Scripts/Player/PlayerScript.cs [hunk 1]
```diff
@@ -78,6 +78,10 @@ public class PlayerScript : ManagedNetworkBehaviour, IMatrixRotation, IAdminInfo
 
 	public float RTT;
 
+	[SyncVar]
+	public bool RcsMode;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Player/PlayerScript.cs:83]: this one will be serialized and appear in editor. I assume this shouldn't be settable in inspector, use either ``[HideInInspector]`` or ``[NonSerializable]`` depending on which is best for this case.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Player/PlayerScript.cs:83]: humm, michalczemierowski is not really the best at adding this stuff?
should I merge it?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Player/PlayerScript.cs:83]: Can you suggest and commit? I’m on my phone now. Then we can merge

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Player/PlayerScript.cs:83]: ok cool

Comment (michalczemierowski) [reply UnityProject/Assets/Scripts/Player/PlayerScript.cs:83]: Sorry, I must have missed it

**Pull request merged**